WordPress/src/main/java/org/wordpress/android/models/news/NewsItemType.kt

```diff
@@ -0,0 +1,20 @@
+package org.wordpress.android.models.news
+
+import org.wordpress.android.R.string
+
+enum class NewsItemType(
```

Comment:
idk if this makes that much sense as an enum. Do you expect more types with fixed resources? If not, what about creating object like this instead? 
```
object LocalNewsItem {
    val titleResId: Int = string.news_card_announcement_title_sample_announcement
    val contentResId: Int = string.news_card_announcement_content_sample_announcement
    val actionResId: Int = string.news_card_announcement_action_sample_announcement
    val urlResId: Int = string.news_card_announcement_action_url_sample_announcement
    val version: Int = -1
}
```

Reply:
Good tip;). 👍